src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -43,6 +42,12 @@ public virtual void ConfigureServices(IServiceCollection services)
                 fhirServerBuilder.AddSqlServer(Configuration);
             }
 
+            /*
+            IHostedServices are executed depends on the order they are added to the DI Container.
+            So need to ensure that the schema is initialized before the background workers are started
+            */
```

Comment:
Nit: slight rewording - what do you think of this?

`The execution of IHostedServices depends on the order they are added to the dependency injection container, so we need to ensure that the schema is initialized before the background workers are started.`

I usually opt for typing out abbreviations instead of shortening them for clarity. Ideally someone who is looking at `Startup.cs` would know that DI = "dependency injection", but I still think it's nice to avoid short forms.

Reply:
I also like `.` in the end of sentence :)

---
In reply to: [518921356](https://github.com/microsoft/fhir-server/pull/1429#discussion_r518921356) [](ancestors = 518921356)